### ISSUE TITLE
Breadcrumb is always 'current' except first load even with ':exact' matching

### ISSUE THREAD

**Issue by TheNeikos** [piotrmurach/loaf#13]
I am not sure how to explain it better, but basically this is what is going on:

- call breadcrumb for the equivalent of '/user/favorites/5' with `match: :exact` on the controller
- call breadcrumb for the equivalent '/user/favorites/5/edit' in the handler method

The first time this is loaded the first is not set 'current', which is correct.
However on subsequent loads it is then reported as 'current' which is incorrect.

I have tried disabling turbolinks, but that didn't change anything.

The relevant bits of code in my controller:

```
breadcrumb ->(c) { "#{c.find_user(c.params[:user_id]).username}'s Favorites" }, :user_favorite_folders, match: :exclusive

def show
  @favorite_folder = authorize find_favorite_folder(params[:id])
  breadcrumb @favorite_folder.title, :user_favorite_folder
end
```

I then render it with:

```
<section class="container is-fluid">
  <nav aria-label="breadcrumbs" class="breadcrumb">
    <ol>
      <% breadcrumb_trail do |crumb| %>
        <li class="<%= "is-active" if crumb.current? %>"><%= link_to crumb.name, crumb.url, crumb.current? ? {'aria-current' => 'page' } : {}) %></li>
      <% end %>
    </ol>
  </nav>
</section>
```

Here `current?` seems to return true even if the url is not exactly matching, but only on subsequent page loads after application start.

I am using:

Rails 5.1
Loaf 0.6

TL;DR:

- On first page load `current?` returns `false`, which is correct
- On any page load afterwards it returns `true`, which is not correct in this case

Am I using it worng?

**Comment by piotrmurach** [comment]
Released `v0.6.1` with the fix!

**Comment by TheNeikos** [comment]
Awesome! Thanks
